src/pages/Dashboard/Reviews/Detail/index.jsx

```diff
@@ -0,0 +1,360 @@
+
+import { Image, Button } from 'components/Base';
+import Layout, {
+  Grid, Row, Section, Card, Dialog
+} from 'components/Layout';
+import Status from 'components/Status';
+import AppName from 'components/AppName';
+import DetailTabs from 'components/DetailTabs';
+import CheckFiles from 'components/CheckFiles';
+
+import styles from './index.scss';
+
+const tabs = [
+  { name: 'Base Info', value: 'baseInfo' },
+  { name: 'Instructions', value: 'readme' },
+  { name: 'Terms of service', value: 'service' },
+  { name: 'Config File', value: 'configFile' },
+  { name: 'Set Price', value: 'price', isDisabled: true },
+  { name: 'Update Log', value: 'updateLog' }
+];
+
+@translate()
+@inject(({ rootStore }) => ({
+  rootStore,
+  appVersionStore: rootStore.appVersionStore,
+  appStore: rootStore.appStore,
+  categoryStore: rootStore.categoryStore,
+  user: rootStore.user
+}))
+@observer
+export default class ReviewDetail extends Component {
+  async componentDidMount() {
+    const { appStore, appVersionStore, match } = this.props;
+    const { appId, versionId } = match.params;
+
+    await appStore.fetch(appId);
+    await appVersionStore.fetch(versionId);
+    await appVersionStore.fetchReviewDetail(appId, versionId);
+  }
+
+  changeTab = tab => {
+    const { appStore } = this.props;
+    appStore.detailTab = tab;
+  };
+
+  handleVersion = async (handleType, versionId) => {
+    const { appVersionStore } = this.props;
+
+    await appVersionStore.handle(handleType, versionId);
+  };
+
+  showReasonDialog = () => {
+    const { appVersionStore } = this.props;
+    appVersionStore.reason = '';
+    appVersionStore.isDialogOpen = true;
+  };
+
+  submitReason = async () => {
+    const { appVersionStore, t } = this.props;
+    const { handle, version, reason } = appVersionStore;
+
+    if (!reason) {
+      appVersionStore.error(t('Please input reject Reason'));
+    } else {
+      await handle('reject', version.version_id);
+    }
+  };
+
+  renderReasonDialog = () => {
+    const { appVersionStore, t } = this.props;
+    const {
+      isDialogOpen, hideModal, changeReason, reason
+    } = appVersionStore;
+
+    return (
+      <Dialog
+        title={t('Reject reason')}
+        isOpen={isDialogOpen}
+        onCancel={hideModal}
+        onSubmit={this.submitReason}
+      >
+        <textarea
+          className={styles.reason}
+          onChange={changeReason}
+          maxLength={500}
+        >
+          {reason}
+        </textarea>
+      </Dialog>
+    );
+  };
+
+  renderVersionReview() {
+    const { appStore, appVersionStore, t } = this.props;
+    const { version, reviewDetail } = appVersionStore;
+    const { phase } = reviewDetail;
+    const phaseKeys = _.keys(phase);
+
+    return (
+      <div className={styles.versionReview}>
+        <div className={styles.title}>{t('审核进度')}</div>
+        <Card className={styles.submit}>
+          {t('Submit')}
+          <label className={styles.time}>2018年10月19日 23:32:22</label>
```

Comment:
hard code time ?

Reply:
There is no api data here, later integrated api will  refactor.

Reply:
Allright